frappe/www/logout.html

```diff
@@ -0,0 +1,25 @@
+{% from "frappe/templates/includes/avatar_macro.html" import avatar %}
+{% extends "templates/web.html" %}
+{% block navbar %}{% endblock %}
+{% block footer %}{% endblock %}
+{% block page_content %}
+<script>
+    window.onload = function() {
+        frappe.call({
+			method: "logout",
+			callback: function (r) {
+				if (r.exc) {
+					return;
+				}
+				redirect_to_login();
+			},
+        });
+
+        function redirect_to_login(){
+            window.location.href = `/login?redirect-to=${encodeURIComponent(
```

Comment:
If you click logout while on `/me`, this happens, resulting in a logout loop

![image](https://github.com/user-attachments/assets/9c5af124-d662-414c-a543-b7ec2f2d406f)

Reply:
Also seems like pre-commit doesn't format this, could you see if you can keep the script in a separate JS file, otherwise do fix the formatting here manually / with IDE.